### PR TITLE
test: add ramda placeholder and make tests consistent

### PR DESCRIPTION
**Pull request by js636f**
Batch 21

Files:

test/
- padCharsStart.js
- padEnd.js
- padStart.js
- pathNotEq.js
- pathOrLazy.js
- paths.js
- pickIndexes.js
- propNotEq.js
- reduceIndexed.js

### CODE REVIEW

File: test/pathNotEq.js [hunk 1]
```diff
@@ -10,22 +10,15 @@ describe('pathNotEq', function() {
...
-    assert.isTrue(RA.pathNotEq(['a', 'b'], 'foo', obj));
-    assert.isTrue(RA.pathNotEq(['a', 'b'])('foo', obj));
-    assert.isTrue(RA.pathNotEq(['a', 'b'], 'foo')(obj));
-    assert.isTrue(RA.pathNotEq(['a', 'b'])('foo')(obj));
-  });
-
-  it('tests path value is not equal', function() {
+  it('should return true if given path values are not equal', function() {
     assert.isTrue(RA.pathNotEq(['a', 'b'], 'foo', obj));
   });
```
Comment (char0n) [inline test/pathNotEq.js]: Can we break this into `context/specify` ? THanks

```js
context('given path values are equal', function() {
  specify('should return false', function() {
  ...assertions...  
  });
});

File: test/paths.js [hunk 2]
```diff
@@ -5,30 +5,30 @@ import * as RA from '../src';
 describe('paths', function() {
   const obj = { a: { b: { c: 1 } }, d: 4, e: 5, f: 6 };
 
-  it('returns empty array if no paths requested', function() {
+  it('should return an empty array if no paths requested', function() {
```
Comment (char0n) [inline test/paths.js]:
```js
context('given no paths requested', function() {
  specify('should return an empty array', function() {
  ...assertions...  
  });
});
```

File: test/paths.js [hunk 3]
```diff
@@ -5,30 +5,30 @@ import * as RA from '../src';
...
+  it('should preserve order', function() {
     assert.sameOrderedMembers(RA.paths([['f'], ['a', 'b', 'c'], ['e']], obj), [
       6,
       1,
       5,
     ]);
   });
 
-  it('returns undefined for nonexistent paths', function() {
+  it('should return undefined for nonexistent paths', function() {
```
Comment (char0n) [inline test/paths.js]:
```js
context('given non-existent paths', function() {
  specify('should return undefined', function() {
  ...assertions...  
  });
});
```

File: test/paths.js [hunk 4]
```diff
@@ -5,30 +5,30 @@ import * as RA from '../src';
...
     const ps = RA.paths([['d'], ['nonexistent']], obj);
     assert.lengthOf(ps, 2);
     assert.strictEqual(ps[0], 4);
     assert.isUndefined(ps[1]);
   });
 
-  it('is curried', function() {
+  it('should be curried', function() {
     assert.sameOrderedMembers(RA.paths([['a', 'b', 'c'], ['d']])(obj), [1, 4]);
```
Comment (char0n) [inline test/paths.js]: Let's add one more assertion
```js
assert.sameOrderedMembers(RA.paths([['a', 'b', 'c'], ['d']])(obj), [1, 4]);
assert.sameOrderedMembers(RA.paths([['a', 'b', 'c'], ['d']], obj), [1, 4]);
```

File: test/propNotEq.js [hunk 5]
```diff
@@ -10,22 +10,15 @@ describe('propNotEq', function() {
...
-    assert.isTrue(RA.propNotEq('a')('foo')(obj));
-  });
-
-  it('tests prop value is not equal', function() {
+  it('should return true if given props values are not equal', function() {
     assert.isTrue(RA.propNotEq('a', 'foo', obj));
   });
 
-  it('tests prop value is equal', function() {
+  it('should return false if given props values are equal', function() {
```
Comment (char0n) [inline test/propNotEq.js]:
```js
context('given props values are equal', function() {
  specify('should return false', function() {
  ...assertions...  
  });
});
```

File: test/reduceIndexed.js [hunk 6]
```diff
@@ -4,74 +4,33 @@ import * as R from 'ramda';
...
+  it('should fold simple functions over arrays with the supplied accumulator', function() {
+    assert.strictEqual(RA.reduceIndexed(addition, 0, [1, 2, 3, 4]), 10);
+    assert.strictEqual(RA.reduceIndexed(mult, 1, [1, 2, 3, 4]), 24);
+  });
 
-      const xf = R.take(2);
-      const apendingT = {};
-      apendingT['@@transducer/result'] = R.identity;
-      apendingT['@@transducer/step'] = R.flip(R.append);
+  it('should ispatch to objects that implement `reduce`', function() {
```
Comment (char0n) [inline test/reduceIndexed.js]:
```suggestion
  it('should dispatch to objects that implement `reduce`', function() {
```

File: test/reduceIndexed.js [hunk 7]
```diff
@@ -101,4 +60,8 @@ describe('reduceIndexed', function() {
       );
     });
   });
+
+  it('should be curried', function() {
+    assert.strictEqual(RA.reduceIndexed(addition)(0)([1, 2, 3, 4]), 10);
```
Comment (char0n) [inline test/reduceIndexed.js]: Let's add some more assertions. Thanks!
```suggestion
    assert.strictEqual(RA.reduceIndexed(addition, 0, [1, 2, 3, 4]), 10);
    assert.strictEqual(RA.reduceIndexed(addition)(0, [1, 2, 3, 4]), 10);
    assert.strictEqual(RA.reduceIndexed(addition, 0)([1, 2, 3, 4]), 10);
    assert.strictEqual(RA.reduceIndexed(addition)(0)([1, 2, 3, 4]), 10);
```

File: test/pathNotEq.js (hunk 1 above)
Comment (js636f) [reply test/pathNotEq.js]: Yes, of course.

**Comment by js636f** [comment]
Thank you! CircleCI failed again, I'll keep restarting it until all tests will be green.

**Pull request merged**